src/components/filter/Filter.jsx

```diff
@@ -0,0 +1,102 @@
+import "./filter.css";
+export const Filter = ({ dispatch, state }) => {
+  const { includeOutOfStock } = state;
+  return (
+    <aside className="filter box-shadow">
+      <div className="filter-heading-container border-bottom">
+        <h4>FILTERS</h4>
+        <a href="" className="text-style-none color-red">
+          CLEAR
+        </a>
+      </div>
+      <div className="sorting-container border-bottom">
+      <h4 className="txt-align-left">Sort Price</h4>
+        <input className="place-left"
+          type="radio"
+          name="sortData"
+          checked={state.sortBy === "LOW_TO_HIGH"}
+          onChange={() => dispatch({ type: "LOW_TO_HIGH" })}
+        />
+        <label>Low To High</label>
+        <br />
+        <input className="place-left"
+          type="radio"
+          name="sortData"
+          checked={state.sortBy === "HIGH_TO_LOW"}
+          onChange={() => dispatch({ type: "HIGH_TO_LOW" })}
+        />
+        <label>High To Low</label>
```

Comment:
the label tag should be linked with their respective input tag, so that if a user clicks on the label, the input field also gets checked.

Reply:
done.